doc/Scorer.rst

```diff
@@ -36,12 +38,15 @@ Else you have to build `KenLM <https://github.com/kpu/kenlm>`_ first and then pa
       --binary_a_bits 255 --binary_q_bits 8 --binary_type trie
 
 
-Afterwards you can use ``generate_package.py`` to generate the scorer package using the ``lm.binary`` and ``vocab-500000.txt`` files:
+Afterwards you can use ``generate_scorer_package`` to generate the scorer package using the ``lm.binary`` and ``vocab-500000.txt`` files:
 
 .. code-block:: bash
 
     cd data/lm
-    python3 generate_package.py --alphabet ../alphabet.txt --lm lm.binary --vocab vocab-500000.txt \
+    # Download and extract appropriate native_client package:
+    curl -LO ...
```

Comment:
what are those `...` ? Leftover ? Maybe we should at least use URL that looks like valid to make it easier and remind people that they can download from our github releases page